auth/grants.go

```diff
@@ -69,8 +105,6 @@ type VideoGrant struct {
 	RoomAdmin bool   `json:"roomAdmin,omitempty"`
 	RoomJoin  bool   `json:"roomJoin,omitempty"`
 	Room      string `json:"room,omitempty"`
```

Comment:
There is already `room` in `VideoGrant`. Would top-level `room` cause confusion?

Reply:
good point.. i'll rename to `roomConfig`